maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -100,7 +99,6 @@ class IOSDriver(
             .getOrThrow {
                 MaestroException.UnableToLaunchApp("Unable to launch app $appId ${it.message}")
             }
-        waitForAppToSettle(null)
```

Comment:
I believe this is required

Reply:
I've added this line to try and solve the scrolling issue. Now that it's solved a different way, the line can go.
It is more efficient to wait Before running a command when needed instead of waiting after a command because some commands don't need the view to be in quiescence state.